lib/animas/animasSimulator.js

```diff
@@ -85,9 +90,53 @@ exports.make = function(config){
     events.push(e);
   }
 
+  function checkIfBasalSuspendedByAlarm(alarm,basal) {
+    if ((basal !== null) && (alarm != null) &&
+        (basal.rate === 0) && (basal.deliveryType !== 'suspend')) {
+
+      // Animas does not generate suspend/resume events for alarms,
+      // so we check if an alarm occurred recently or at the same time.
+      // If so, we mark the basal as suspended and generate a new suspend event.
+
+      var duration = Date.parse(basal.time) - Date.parse(alarm.time);
+      console.log(alarm, basal);
```

Comment:
Remove this logging? Or switch to using bows and/or put it behind __DEBUG__ if you think you want to keep it indefinitely for debugging purposes?